docs/INSTRUCTIONS.md

```diff
@@ -3,19 +3,19 @@
 # INSTRUCTIONS FILE FOR NASA's PUBLIC GITHUB ORGANIZATION
 
 ## Purpose
-This GitHub organization is intended to publicly host NASA code that has been SRA-approved for open source. You can review the SRA (software release authority) process for getting open-source approval on https://code.nasa.gov/#/guide.
+This GitHub organization is intended to publicly host NASA code that has been SRA-approved for open source. You can review the SRA (software release authority) process for getting open-source approval on [https://code.nasa.gov/#/guide](https://code.nasa.gov/#/guide).
```

Comment:
Following the pattern set by you, here is only: `[code.nasa.gov/#/guide]`.

Your default was put all url's without `https://`.

I hope that would be better :)

Reply:
Oh , sorry .
I don't seen :|